contents/docs/surveys/_snippets/flutter-surveys-installation.mdx

```diff
@@ -0,0 +1,83 @@
+## Step one: Add PostHog to your app
+
+> 🚧 **Note:** Mobile surveys for Flutter is currently in **beta** and only available for **iOS**. We'd love to hear your feedback on betas via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asurveys%3Alow%3Atrue) or one of our other [support options](/docs/support-options).
+>
+> Using it requires PostHog's Flutter SDK version >= 5.1.0, but it's recommended to always use the latest version.
+
+import FlutterInstall from "../../integrate/_snippets/install-flutter.mdx"
+
+<FlutterInstall />
+
+## Step two: Enable surveys in your configuration
```

Comment:
if you look at the session recording docs, you can see that it requires the `com.posthog.posthog.AUTO_INIT` to be disabled.
I think for surveys is the same thing, right? if that's the case, we should add this note and instructions as part of this step as well

Reply:
worth checking with docs team on how to do this, so we dont duplicate more snippets and setup again
not a blocker for this PR